docs/start/quickstart-exit.md

```diff
@@ -92,7 +92,38 @@ It needs to be the validator client that is connected to your charon client taki
         </ul>
         <pre>
           <code>
-            {String.raw`docker exec -it charon-distributed-validator-node-lodestar-1 /bin/sh -c 'node /usr/app/packages/cli/bin/lodestar validator voluntary-exit --beaconNodes="http://charon:3600" --dataDir=/opt/data --exitEpoch=162304 --network=goerli --yes'`}
+            {String.raw`docker exec -it charon-distributed-validator-node-lodestar-1 /bin/sh -c 'node /usr/app/packages/cli/bin/lodestar validator voluntary-exit \
+            --beaconNodes="http://charon:3600" \
+            --dataDir=/opt/data \
+            --exitEpoch=162304 \
+            --network=goerli \
+            --yes'`}
+          </code>
+        </pre>
+      </TabItem>
+      <TabItem value="charon" label="Charon" default>
+      Voluntary exit can be submitted directly through charon as well. The partially signed exit messages are stored (centrally) on Obol's infrastructure. Exits through charon are submitted per validator. All active validators' public keys can be listed with:
+        <pre>
+          <code>
+        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit active-validator-list'`}
+          </code>
+        </pre>
+      Then a signed partial exit for validator can be submitted by:
+        <pre>
+          <code>
+        {String.raw`docker exec -it charon-distributed-validator-node-charon-1 /bin/sh -c 'charon exit sign \
+    --beacon-node-endpoints="http://lighthouse:5052" \
+    --validator-public-key="<VALIDATOR_PUBLIC_KEY>" \
+    --publish-timeout="5m"'`}
+          </code>
+        </pre>
+      After the threshold of signed partial exits is met, any operator from the cluster can broadcast the full signed exit to the beacon chain:
```

Comment:
While technically correct, we might want to be a little more explicit with "threshold of signed partial exits is met. A user doesn't know that threshold happens when threshold operators broadcasted their partial exit :D